sys/arm64/include/intr.h

```diff
@@ -42,10 +43,16 @@ arm_irq_memory_barrier(uintptr_t irq)
 {
 }
 
+#endif	/* !LOCORE */
+
 #ifdef DEV_ACPI
 #define	ACPI_INTR_XREF	1
 #define	ACPI_MSI_XREF	2
 #define	ACPI_GPIO_XREF	3
 #endif
```

Comment:
I object to this since these will almost certainly **never** be used by assembly language.  I understand the instinct to make as much as possible available to assembly language, but the ACPI constants are too improbable to merit being made available.